plugins/scanners/fossid/src/main/kotlin/OrtComment.kt

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (C) 2023 The ORT Project Authors (see <https://github.com/oss-review-toolkit/ort/blob/main/NOTICE>)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.plugins.scanners.fossid
+
+import org.ossreviewtoolkit.model.TextLocation
+
+/**
+ * A class representing a comment generated by ORT and attached to a file marked as identified in FossID.
+ * This comment contains a mapping between license identifiers and their corresponding source locations.
+ * This comment is serialized as JSON in the marked as identified file's comment, hence the property being named
+ * [OrtComment.ort].
+ */
+data class OrtComment(val ort: OrtCommentPayload)
+
+/**
+ * The payload of an [OrtComment].
+ */
+data class OrtCommentPayload(
+    /**
+     * The license of chosen snippets mapped to their source location.
+     */
+    val licenses: Map<String, List<TextLocation>>,
+    /**
+     * The amount of chosen snippets for this file.
+     */
+    val snippetChoiceCount: Int,
+    /**
+     * The amount of non-relevant snippets for this file.
+     */
+    val nonRelevantSnippetsCount: Int
```

Comment:
`notRelevantSnippetCount` for consistency?

Reply:
With a 's' in `notRelevantSnippetsCount` right ?

Reply:
I have a tendency for the singular but both works. Just go with the same as above where it was about `Choice`/`Choices`.

Reply:
Ok let's go with `notRelevantSnippetsCount`.